go-controller/hybrid-overlay/pkg/controller/node_linux_test.go

```diff
@@ -386,28 +395,22 @@ var _ = Describe("Hybrid Overlay Node Linux Operations", func() {
 			Expect(err).NotTo(HaveOccurred())
 
 			addEnsureHybridOverlayBridgeMocks(nlMock)
-			err = n.controller.EnsureHybridOverlayBridge(node)
-			Expect(err).NotTo(HaveOccurred())
 
-			linuxNode, okay := n.controller.(*NodeController)
-			Expect(okay).To(BeTrue())
-			Expect(linuxNode.initialized).To(BeTrue())
-
-			// ovs commands generated by the initial sync
-			fexec.AddFakeCmdsNoOutputNoError([]string{
-				"ovs-ofctl dump-flows --no-stats br-ext table=20",
-				"ovs-ofctl -O OpenFlow13 --bundle replace-flows br-ext -",
-			})
+			// initial flowSync
+			addSyncFlows(fexec)
+			// flowsync after EnsureHybridOverlayBridge()
+			addSyncFlows(fexec)
 
-			//  perform the requested cacheSync
-			linuxNode.syncFlows()
+			f.Start(stopChan)
+			go func() {
+				n.Run(stopChan)
```

Comment:
@JacobTanenbaum you should wg.Add(1) before the go func() and then defer wg.Done() inside it to make sure these are terminated when the next test starts.

Reply:
done